superflore/generators/bitbake/ros_meta.py

```diff
@@ -17,16 +17,15 @@
 
 from superflore.repo_instance import RepoInstance
 from superflore.utils import info
-from superflore.utils import rand_ascii_str
 
 
 class RosMeta(object):
     def __init__(
-       self, repo_dir, do_clone, org='ros', repo='meta-ros', from_branch=''
+        self, dir, do_clone, branch, org='ros', repo='meta-ros', from_branch=''
```

Comment:
strange that the linter didn't catch that indentation issue... Thanks for fixing that.